sp800-63a/sec4_ial.md

```diff
@@ -6,9 +6,9 @@
 
 _This section is normative._
 
-The paradigm of this document is that individuals (referred to as applicants at this stage) undergo an identity proofing and enrollment process in which their identity evidence and attributes are collected, uniquely resolved to a single identity record, then validated and verified. These attributes are then bound to an authenticator (described in [SP 800-63B](#800-63b)).
+This document describes the common pattern in which a subject (referred to as an applicant at this stage) undergoes an identity proofing and enrollment process in which their identity evidence and attributes are collected, uniquely resolved to a single identity record, then validated and verified. A CSP may then bind these attributes to an authenticator (described in [SP 800-63B](#800-63b)).
```

Comment:
"single identity record": Might we want to introduce the term credential here? (not essential)